pelican/__init__.py

```diff
@@ -455,6 +455,17 @@ def parse_arguments(argv=None):
         ),
     )
 
+    LOGS_HANDLERS = {"plain": None, "rich": DEFAULT_LOG_HANDLER}
+    parser.add_argument(
+        "--logs-handler",
+        default="rich",
+        choices=LOGS_HANDLERS,
+        help=(
+            "Which handler to to format log messages. "
```

Comment:
I think the word "use" is missing here? I assume this was intended to say:

> "Which handler to use to format log messages. "

Reply:
Woops.  Thanks, fixed!